CMakeLists.txt

```diff
@@ -65,7 +65,7 @@ macro(build_console_bridge)
     DIRECTORY
       ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_install/
     DESTINATION
-      ${CMAKE_INSTALL_PREFIX}
+      share/${PROJECT_NAME}
```

Comment:
This changes the semantic which I assume is undesired. Have you compared the location of the installed files before and after this change?

Reply:
I am pretty sure you want to use `.` here.

Reply:
@Karsten1987 FYI `shared_queues_vendor`, `sqlite3_vendor` and `zstd_vendor` all have the same bug.

Reply:
Oh, good point.  It still seems to work, but no reason to make a change here if not necessary.  Let me poke at it just a bit.

Reply:
If I switch it to `.`, then the unwanted fully-qualified path comes back in the class_loaderExport.cmake file.  If I switch it to `share/${PROJECT_NAME}/../..`, the unwanted fully -qualified path comes back.  Does CMake have some kind of detection for relative paths like this and rewrite them with an absolute?

Reply:
From the docs of `install(DESTINATION ...)`:

> If a full path (with a leading slash or drive letter) is given it is used directly. If a relative path is given it is interpreted relative to the value of the `CMAKE_INSTALL_PREFIX` variable.

Reply:
Now I'm thinking that this is actually a bug in console_bridge itself.  Specifically, I think that this line is probably wrong: https://github.com/ros/console_bridge/blob/2992a853feacaf7f3d269b3e7be0cbdbbc993543/CMakeLists.txt#L84 (for the record, this was added in https://github.com/ros/console_bridge/pull/45).

The [documentation](https://cmake.org/cmake/help/v3.10/module/CMakePackageConfigHelpers.html) for configure_package_config_file() strongly implies that you should not use absolute paths here, but we currently are using one there.

I've run out of time for this today, but I'll poke at it more tomorrow.